addons/website_crm_score/controllers/main.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+from openerp import addons, http, SUPERUSER_ID, fields
+from openerp.http import request
+
+
+class PageController(addons.website.controllers.main.Website):
+
+    @http.route('/page/<page:page>', auth="public", website=True)
+    def page(self, page, **opt):
+        response = super(PageController, self).page(page, **opt)
+        # duplication of ir_http.py
+
+        if hasattr(response, 'status_code') and response.status_code == 200:
+            try:
+                view = request.website.get_template(page)
+                if view.track:  # avoid tracking redirected page
+                    cr, uid, context = request.cr, request.uid, request.context
+                    lead_id = request.registry["crm.lead"].decode(request)
+                    url = request.httprequest.url
+                    vals = {'lead_id': lead_id, 'partner_id': request.session.get('uid', None), 'url': url}
+
+                    if lead_id and request.registry['website.crm.pageview'].create_pageview(cr, uid, vals, context=context):
+                        # create_pageview was successful
+                        pass
+                    else:
+                        response.delete_cookie('lead_id')
+                        request.session.setdefault('pages_viewed', {})[url] = fields.Datetime.now()
+                        request.session.modified = True
+            except:
+                pass  # view not found
```

Comment:
View not found or anything else really. Should probably be `try: [find view] except: pass else: [do stuff if view found]`